client/components/checklist/task.js

```diff
@@ -95,19 +79,31 @@ export class ChecklistTask extends PureComponent {
 						</small>
 					) }
 				</div>
-				<Focusable
+				<ToggleContainer
 					className="checklist__task-icon"
-					onClick={ this.handleToggle }
-					aria-pressed={ completed ? 'true' : 'false' }
+					onClick={ onDismiss }
+					aria-pressed={
+						/* eslint-disable-next-line no-nested-ternary */
+						onDismiss ? ( completed ? 'true' : 'false' ) : undefined
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/26216/files#r208360821 :grimacing:

Reply:
Applied your suggestion in 540e0bdd7852a93c310ee7a5e1169e62aefea194. I'm a bit indifferent as to the clearer approach so we can leave whatever you think is best.

> Do we really need the bool strings ('true'/'false') here?

I believe this is the correct way to do it. It is how official docs show it: https://reactjs.org/docs/accessibility.html#wai-aria

Note that these are `"true"`/`"false"` strings in the DOM, a bit different from some other boolean-like DOM attributes, e.g. `disabled`.